t/mojo/dom.t

```diff
@@ -2643,10 +2643,39 @@ subtest 'Comments' => sub {
 <!-- HTML4 -- >
 <!-- bad idea -- HTML4 -- >
 EOF
-  is $dom->tree->[1][1], ' HTML5 ',             'right comment';
-  is $dom->tree->[3][1], ' bad idea -- HTML5 ', 'right comment';
-  is $dom->tree->[5][1], ' HTML4 ',             'right comment';
-  is $dom->tree->[7][1], ' bad idea -- HTML4 ', 'right comment';
+  is $dom->tree->[1][1], ' HTML5 ',                      'right comment';
+  is $dom->tree->[3][1], ' bad idea -- HTML5 ',          'right comment';
+  is $dom->tree->[5][1],  '<',                           'not support html style comments';
+  is $dom->tree->[6][1], "!-- HTML4 -- >\n",             'not support html style comments';
+  is $dom->tree->[7][1], '<',                            'not support html style comments';
+  is $dom->tree->[8][1], "!-- bad idea -- HTML4 -- >\n", 'not support html style comments';
+
+  # Issue #2030
```

Comment:
We don't use comments like that.

Reply:
You mean: `# Issue #2030`?

No problem. I can remove it